integration_tests/test_gov_update_params.py

```diff
@@ -0,0 +1,88 @@
+from pathlib import Path
+import json
+
+import pytest
+from web3 import Web3
+
+from .network import setup_custom_cronos
+from dateutil.parser import isoparse
+from .utils import (
+    ADDRS,
+    CONTRACTS,
+    KEYS,
+    deploy_contract,
+    send_txs,
+    sign_transaction,
+    wait_for_new_blocks,
+    wait_for_block_time,
+    parse_events,
+)
+
+
+@pytest.fixture(scope="module")
+def cronos_gov(tmp_path_factory):
+    path = tmp_path_factory.mktemp("cronos-gov")
+    yield from setup_custom_cronos(
+        path, 26300, Path(__file__).parent / "configs/default.jsonnet"
```

Comment:
You can use the existing cronos fixture.

Reply:
directly using `cronos` like this:
```python
def test_gov_update_params(cronos, tmp_path):
...
```

Reply:
done